app/moves/constants.js

```diff
@@ -0,0 +1,10 @@
+const CONSTANTS = {
```

Comment:
Should we have a noddy tautological test to ensure that if anyone changes these values, they at the very least have to stop and fix the test and maybe think whether there are any conserquences of changing it?

Reply:
So in theory we don't actually care in terms of the unit test I don't think.

That's probably something that we should do with the e2e tests. Ie, if I click the tab in the navigation, by default I see View X.

But unfortunately those tests are still WIP... :/